packages/amplify-prompts-vNext/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@aws-amplify/amplify-prompts",
+  "version": "2.6.5",
+  "description": "Utility functions for Amplify CLI terminal I/O",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "clean": "rimraf lib tsconfig.tsbuildinfo node_modules",
+    "demo": "yarn build && node lib/demo/demo.js",
+    "test": "jest --logHeapUsage --color",
+    "watch": "tsc -w",
+    "extract-api": "ts-node ../../scripts/extract-api.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aws-amplify/amplify-cli.git"
+  },
+  "keywords": [
+    "amplify",
+    "cli",
+    "prompts"
+  ],
+  "author": "Amazon Web Services",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/aws-amplify/amplify-cli/issues"
+  },
+  "homepage": "https://github.com/aws-amplify/amplify-cli#readme",
+  "dependencies": {
+    "amplify-prompts": "2.6.5"
+  },
```

Comment:
in next iteration we're going to prefix actual `amplify-prompts` and delete this shell.